libs/core/pyproject.toml

```diff
@@ -77,8 +77,42 @@ target-version = "py39"
 
 
 [tool.ruff.lint]
-select = [ "AIR", "ANN", "ASYNC", "B", "C4", "COM", "DJ", "E", "EM", "EXE", "F", "FA", "FAST", "FLY", "FURB", "G", "I", "ICN", "INP","INT", "ISC", "LOG", "N", "NPY", "PD", "PIE", "PLW", "PTH", "Q", "RSE", "S", "SIM", "SLOT", "T10", "T20", "TC", "TID", "TRY", "UP", "W", "YTT",]
-ignore = [ "ANN401", "COM812", "FA100", "ISC001", "S110", "S112", "TC001", "TC002", "TC003", "UP007",]
+select = [ "ALL",]
+ignore = [
+    "C90", # McCabe complexity
```

Comment:
I propose to ignore complexity checks as it can be a burden to follow.
Can be reconsidered later if needed.